docs/tsc/meetings/2020-07-20.md

```diff
@@ -0,0 +1,110 @@
+      we have been discussing the best way to merge. DCO issus created some 
+      branch conflicts earlier so I resubmitted from a clean branch. Sean could 
+      follow with a supplemental PR if we merge it now.
+    - Sean: Patrick and I encountered issues with this change discovering 
+      Python 3 on our systems (macOS and Linux respectively). The proposed fix 
+      resolved these problems for both.
+    - Michael: CI is passing, but that may be due to using explicit Python 
+      paths in the CMake commands. Sounds like the current method os not 
+      finding python-dev headers.
+    - **TODO**: Merge the CMake changes as-is since CI is passing, and Sean 
+      will submit fix in a followup PR since the change is small.
+
+* Website design/logo refresh:
+    - Doug: Month out from feature complete. Need the update docs before 
+      inviting people in to v2. Pushing up to the wire. We need to focus on 
+      docs in the coming weeks. Looking at Simran's toolchain data flow 
+      diagram, this provides good overview of documentation approach. Is it 
+      correct?
+    - Sean: Yes, just need to add a node for generating RST from 
+      pybind11 code.
+    - Doug: Can we define concrete projects so contributors can work on this? 
+      In the next two weeks we either need to get this working or stay with 
+      existing system.
+    - Sean: Docstring header PR is in. Need another PR for pybind11 docstring 
+      header, and another for reorganization of doc structure. The RTD page 
+      points to official OCIO repo, so we're live now, but API docs and new 
+      style is missing. Any changes to master are getting pushed to RTD now. 
+      Not as much holding back docs as it seems. 
+    - Simran: An event hook in GH communicates to RTD when to update the site.
+    - Michael: Can we open GH issues for logical chunks of work?
+    - Sean: Remaining infrastructure work is to finish Python docs extraction. 
+      The CMake workflow needs work, which uses another pybind11 component 
+      that doesn't install with pybind11 by default, so will be need to pulled 
+      at runtime. 
+    - **TODO**: Sean will create PR with his current Python extraction work. 
+      Chin-Ying Li willing to assist with this work to help finish doc 
+      infrastructure.
+    - Doug: There are two parts of docs: infrastructure and writing content. 
+      Separate tasks. Infrastructure should be in place to see what final 
+      product is going to look like. For content objective, Carol has been 
+      writing table of contents/outline. Lots of discussion on that. AD team 
+      can help write docs.
+    - Sean: Would be good to get GH issues and volunteers for writing certain 
+      sections of docs.
+    - **TODO**: Break remaining work into GH issues to help with contribution.
+    - **TODO**: Michael will reach out to LF creative services about help with 
+      frontend work. Can collaborate with community on re-branding effort.
```

Comment:
What's the scope here? Website (GitHub Pages) + documentation or just one of them?
A Sphinx theme in case of the docs?

Reply:
I believe Sean was referencing this script: https://github.com/pybind/pybind11/blob/master/tools/mkdoc.py Used for generating RST from C++ header files.